tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -111,46 +113,99 @@ type stressConfig struct {
 }
 
 // NewStresser creates stresser from a comma separated list of stresser types.
-func NewStresser(s string, sc *stressConfig, m *member) Stresser {
+func NewStresser(s string, sc *stressConfig, members []*member) Stresser {
```

Comment:
I noticed that call NewStresser() for each member will create runner_stresser len(members) times since runner_stresser is not depend on each member but endpoints. How should I get around that?